File: autoscale/src/main/java/com/sequenceiq/periscope/log/LoggerContextKey.java
package com.sequenceiq.periscope.log;

public enum LoggerContextKey {

    RESOURCE_ID("resourceId"),
    CB_STACK_ID("cbStack");

    private final String value;

    LoggerContextKey(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: autoscale/src/main/java/com/sequenceiq/periscope/log/LoggerContextKey.kt
package com.sequenceiq.periscope.log

enum class LoggerContextKey private constructor(private val value: String) {

    OWNER_ID("owner"),
    RESOURCE_ID("resourceId"),
    CB_STACK_ID("cbStack");

    override fun toString(): String {
        return this.value
    }
}
